rtcpeerconnection.js

```diff
@@ -370,8 +372,7 @@ PeerConnection.prototype.offer = function (constraints, cb) {
         function (err) {
```

Comment:
Same comment here as on traceablepeerconnection - I'd pass the error handler to `catch`